File: LES_EXAMENS/EXAM_PYTHON_2025_2026_IA_RATTRAPAGE/exerciceF.py
T=[[None for j in range(10)] for i in range(10)]
for i in range(0,10):
    for j in range(0,10):
        T[i][j]=int(input(f"saisir la case {i+1} {j+1}"))

#calcule et affich la somme des lignes
print("la somme des lignes.")
somme=0
for i in range(0,10):
    for j in range(0,10):
        somme=somme+T[i][j]
    print(somme)
    somme=0
#calcule et affich la somme des colonnes
print("la somme des colonnes.")
somme=0
for j in range(0,10):
    for i in range(0,10):
        somme=somme+T[i][j]
    print(somme)
    somme=0

File: LES_EXAMENS/EXAM_PYTHON_ET_EXAM_ALGO_C__2025_2026_IA_RATTRAPAGE/exerciceF.c
#include <stdio.h>
int i,j,somme;
int T[10][10];
int main(){
    for(int i=0;i<=9;i++){
        for(int j=0;j<=9;j++){
            printf("saisir la case %d %d: ",i+1,j+1);
            scanf("%d",&T[i][j]);
        }
    }
    //calcule et affich la somme des lignes
    printf("la somme des lignes.\n");
    somme=0;
    for(int i=0;i<=9;i++){
        for(int j=0;j<=9;j++){
            somme=somme+T[i][j];
        }
        printf("%d\n",somme);
        somme=0;
    }
    //calcule et affich la somme des colonnes
    printf("la somme des colonnes.\n");
    somme=0;
    for(int j=0;j<=9;j++){
        for(int i=0;i<=9;i++){
            somme=somme+T[i][j];
        }
        printf("%d\n",somme);
        somme=0;

    }
    return 0;
}